content/articles/2021-06-06-poliastro-google-summer-of-code-2021.md

```diff
@@ -0,0 +1,24 @@
+Title: GSoC 2021, the journey begins!
+Date: 2021-06-06 11:00 IST
+Category: GSOC
+Tags: GSOC, GSOC21, poliastro
+slug: 2021-06-06-poliastro-google-summer-of-code-2021
```

Comment:
Please remove the date from the slug - we've been adding it for some time, but it's redundant :) See the last published article

Reply:
Thanks!